ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -744,6 +744,8 @@ ethernet_interfaces:
     logging:
       event:
         link_status: < true | false >
+    lldp_transmit: < true | false >
```

Comment:
lldp has other options we might implement later. So please refactor to
```yaml
lldp:
  receive: < true | false >
  transmit: < true | false >
```

Reply:
Fixed.